src/SprykerEco/Zed/Econda/Business/Collector/AbstractDatabaseCollector.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Collector;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Service\UtilDataReader\Model\BatchIterator\CountableIteratorInterface;
+use Spryker\Service\UtilDataReader\Model\BatchIterator\PdoBatchIterator;
```

Comment:
**bridges are necessary between different modules**

Reply:
@pugach-ivan please check this